samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _power-mgr-sample:
+
+Power management demo
+#####################
+
+Overview
+********
+
+A sample implementation of a power manager app that uses the Zephyr
+power management infrastructure.
+
+This app will cycle through the various power schemes at every call
```

Comment:
How about:

     This app cycles through the following power state schemes each time the
     _sys_soc_suspend() hook function is called:

Reply:
Is it really a "cycle", or does it stop after three calls?